tests/integration-tests/tests/tests-test.js

```diff
@@ -233,6 +233,12 @@ describe('the tests api', function() {
                 createTestResponse.statusCode.should.eql(201, JSON.stringify(createTestResponse.body));
                 createTestResponse.body.should.have.only.keys('id', 'revision_id');
 
+                const requestFavBody = simpleTest.test;
+                requestFavBody["is_favorite"] = true;
```

Comment:
I suggest taking all related to is_favorite to new tests

1. Add test without is_favorite param => verify it acts as is_favorite false (not returned when filter applied)
2. Add test with is_favorite true => test getTests and getTest
3. Add test with is_favorite false => test getTests and getTest